libs/filamat/src/shaders/CodeGenerator.cpp

```diff
@@ -83,6 +90,16 @@ utils::io::sstream& CodeGenerator::generateProlog(utils::io::sstream& out, Shade
                 out << "#version 410 core\n\n";
                 out << "#extension GL_ARB_shading_language_packing : enable\n\n";
             }
+            if (v.hasStereo() && stage == ShaderStage::VERTEX) {
```

Comment:
how about just `if (v.hasStereo() && stage == ShaderStage::VERTEX && material.stereoscopicType == StereoscopicType::MULTIVIEW)` ? 

Simliar below?

Reply:
Yeah that was my first code, but I actually changed it in favor of the warning detection in switch. Let me change GLSLPostProcessor.cpp in the same way.